application/core/QuestionTypes/ArrayMultiscale/RenderArrayMultiscale.php

```diff
@@ -295,7 +295,7 @@ public function parseSubquestionsNoDropdown(&$aData)
                 if ($aData['shownoanswer']) {
                     // No answer for accessibility and no javascript (but hide hide even with no js: need reworking)
                     $fname0value = $this->getFromSurveySession($myfname0);
-                    if (!isset($fname0value) || $fname0value == '') {
+                    if ($fname0value !== '0' && empty($fname0value)) {
```

Comment:
I really think it's less clear … why not `(!isset($fname0value) || $fname0value === '')` or better `(is_null($fname0value) || $fname0value === '') {` 

There are only 2 value for no answer : NULL : don't see it or '' empty string : see but don't answer.

Reply:
OK, comment replace